update_locales.py

```diff
@@ -19,14 +30,21 @@ def update_locales(api_key: str, path: Path, format: str) -> None:
     for locale in locales:
         num_translated = locale['progress']['translated']
         num_untranslated = locale['progress']['untranslated']
-        if 100 * num_translated / (num_translated + num_untranslated) <= MIN_PROGRESS_PERCENTAGE:
-            continue
 
         locale_code = locale['code'].replace("-", "_")
+
+        is_poorly_translated = 100 * num_translated / (num_translated + num_untranslated) <= MIN_PROGRESS_PERCENTAGE
+        if is_poorly_translated and locale_code not in FORCE_LANGUAGES:
+            continue
+
         file_name = f"{locale_code}.{format}"
         file_path = path / file_name
 
-        response = session.get(f'https://localise.biz/api/export/locale/{file_name}')
+        params = {}
+        if locale_code in AUTO_FALLBACK:
+            params['fallback'] = 'auto'
```

Comment:
Do I understand correctly that the purpose of the fallback is that those languages `['es'_UY, 'eu_ES', 'ca_ES', 'gl_ES']` default to `es_ES` if there is no string provided? I am assuming the default `es_ES` is defined within localise.biz

Reply:
Yes! You're right, it's exactly like you say. The fallback=auto behaviour is configured in localise.biz